projects/fal/src/fal/api.py

```diff
@@ -797,18 +764,114 @@ def order_schema_object(schema: dict[str, Any]):
             Mark the order of properties in the schema object.
             They can have 'allOf', 'properties' or '$ref' key.
             """
-            if "allOf" in schema:
-                for sub_schema in schema["allOf"]:
-                    order_schema_object(sub_schema)
-            if "properties" in schema:
-                mark_order(schema, "properties")
+            for sub_schema in schema.get("allOf", []):
+                order_schema_object(sub_schema)
+
+            mark_order(schema, "properties")
 
-        for key in spec.get("components", {}).get("schemas") or {}:
```

Comment:
does the "properties" key exist always? seems like we were checking the existence before and now we dont

Reply:
nvm, i saw that you moved the existence check into the `mark_order` function itself